tests/conftest.py

```diff
@@ -22,6 +22,15 @@
 
 HARVEST_SOURCE_URL = os.getenv("HARVEST_SOURCE_URL")
 
+# ignore tests in the functional dir
+collect_ignore_glob = ["functional/*"]
```

Comment:
i figured this was some kind of global pytest looks for. [source](https://docs.pytest.org/en/7.1.x/reference/reference.html#globalvar-collect_ignore_glob)